views.py

```diff
@@ -56,35 +66,14 @@ def download(request, path, rest_call=False, use_async=True, *args, **kwargs):
             raise KeyError('settings must have IRODS_GLOBAL_SESSION set '
```

Comment:
@pkdash This else logic will never get executed in our current setup since we always have IRODS_GLOBAL_SESSION in our settings. I thought about removing this but since it has been there since I started to work on HydroShare and it is harmless, I did not do so. We don't have to bother to log this since it will never happen as long as we have the local_settings.py present, and even though it happens, we will get exception raised anyway. Let's just keep the way it is for now.